Cargo.toml

```diff
@@ -36,6 +37,8 @@ device-selected = []
 fence = []
 smi = []
 
+stm32f107 = ["stm32f1xx-hal/stm32f107", "device-selected"]
+
```

Comment:
remove empty line

Reply:
I think it's better to divide between `stm32f1xx`, `stm32f4xx`, and `stm32f7xx` as it is now